system_tests/system_tests_sync/test_external_accounts.py

```diff
@@ -303,3 +305,42 @@ def test_aws_based_external_account(
                 },
             },
         )
+
+def test_pluggable_external_account(
+    oidc_credentials, service_account_info, dns_access
+):
+    now = datetime.datetime.now()
+    unix_seconds = time.mktime(now.timetuple())
+    expiration_time = (unix_seconds + 1 * 60 * 60) * 1000
```

Comment:
nit: I think expiration time could be more descriptive, e.g. `expires_in_one_hour`